src/utils/linkResolver.js

```diff
@@ -0,0 +1,27 @@
+// Note: This file cannot be written in TypeScript because it is imported by
+// gatsby-config.js.
+
+/** A link resolver function for Prismic.
+ *
+ * Accepts a description of an arbitrary Prismic document of any type and
+ * returns a string URL to the document.
+ *
+ * See https://prismic.io/docs/technologies/link-resolver-javascript for the
+ * available properties on the `doc` argument.
+ */
+const linkResolver = (doc) => {
+  switch (doc.type) {
+    case "home_page":
+      return "/";
```

Comment:
What are you thinking this will look like when we have dynamic urls, like for blog posts?

Reply:
For dynamic URLs, I think we'd want to use the `uid` field on the `doc` object, which Prismic's docs say hold the user-friendly unique ID. I think it'd look something like:

```js
switch (doc.type) {
  case "blog_post":
    return `/blog/${doc.uid}`;
}
```

Gatsby has its own [special support for dynamic pages](https://www.gatsbyjs.com/docs/tutorial/part-seven/) as well, which I haven't used before, but we'll want to make those match the Prismic link resolver.